src/app/control-planes/views/ControlPlaneDetailView.vue

```diff
@@ -13,111 +13,166 @@
+
+            <template v-else>
+              <div
+                class="stack"
+                data-testid="detail-view-details"
               >
-                <ErrorBlock
-                  v-if="error"
-                  :error="error"
-                />
+                <template v-if="globalInsight.meshes.total === 1 && meshInsightsData?.items[0].name === 'default' && globalInsight.dataplanes.standard.total === 0">
+                  <KAlert
+                    v-if="isShowingOnboardingAlert"
+                    appearance="success"
+                    dismiss-type="icon"
+                    data-testid="onboarding-notification"
+                    @closed="isShowingOnboardingAlert = false"
+                  >
+                    <template #alertMessage>
+                      <div class="onboarding-alert-content">
+                        <!-- eslint-disable-next-line vue/no-v-html  -->
+                        <div v-html="t('main-overview.detail.onboarding.message', { name: t('common.product.name') })" />
 
-                <template v-else>
-                  <div class="card-header">
-                    <div class="card-title">
-                      <h2>{{ t('main-overview.detail.zone_control_planes.title') }}</h2>
-
-                      <RouterLink :to="{ name: 'zone-cp-list-view' }">
-                        {{ t('main-overview.detail.health.view_all') }}
-                      </RouterLink>
-                    </div>
-                    <!-- Here we check the length of the zones because if the length is zero then -->
-                    <!-- we show a create button in the empty state for the list therefore we don't need -->
-                    <!-- a repeated button here -->
-                    <div
-                      v-if="can('create zones') && (data?.items?.length ?? 0 > 0)"
-                      class="card-actions"
-                    >
-                      <KButton
-                        appearance="primary"
-                        :to="{ name: 'zone-create-view' }"
-                      >
-                        <AddIcon :size="KUI_ICON_SIZE_30" />
-
-                        {{ t('zones.index.create') }}
-                      </KButton>
-                    </div>
-                  </div>
-                  <ZoneControlPlanesList
-                    data-testid="zone-control-planes-details"
-                    :items="data?.items"
-                  />
+                        <KButton
+                          appearance="primary"
+                          size="small"
+                          class="action-button"
+                          :to="{ name: 'onboarding-welcome' }"
+                        >
+                          {{ t('main-overview.detail.onboarding.get_started_link') }}
+                        </KButton>
+                      </div>
+                    </template>
+                  </KAlert>
```

Comment:
Considering this file is used in different environments, do we need this here if we don't have onboarding installed?

Reply:
Previous to this PR we already do that, and we already check in that component if you are on the homepage

https://github.com/kumahq/kuma-gui/pull/1756/files#diff-c6fc3196605e3b659cfce445494e393e521da228c007c2ae53870e7106da454bR5

Reply:
I added the component to the control-planes service config so it can be removed this way.  I know the alert was functionally already isolated to the home page, but I think this mechanism should be in the view used on the home page and not in ApplicationShell.